src/app/common/components/presentation/text-box/text-box.component.ts

```diff
@@ -1,18 +1,20 @@
-import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { Component, OnInit, Input, Output, EventEmitter, OnChanges, SimpleChanges } from '@angular/core';
 
 @Component({
   selector: 'app-text-box',
   templateUrl: './text-box.component.html',
   styleUrls: ['./text-box.component.scss']
 })
-export class TextBoxComponent implements OnInit {
+export class TextBoxComponent implements OnInit, OnChanges {
+
   @Input() private id: string;
   @Input() private placeholder: string;
   @Input() private encryption: string;
   @Input() private icon: string;
   @Input() private textboxValue: string;
+  ngOnChanges(changes: SimpleChanges): void {}
 
-  @Output() textboxChange: EventEmitter<String> = new EventEmitter<String>();
+  //@Output() textboxChange: EventEmitter<String> = new EventEmitter<String>();
```

Comment:
Do we need still need this even though it is commented out?

Reply:
this has been addressed & removed on my recent push just made now